src/ducks/pin/styles.styl

```diff
@@ -29,6 +29,7 @@
 
 .Pin__button:active
     background-color rgba(255, 255, 255, 0.25)
+    transition background-color 0s
```

Comment:
Nit: did you try to use `transition unset`? If it works, it would be more DRY

Reply:
Done, thanks !